files/br.make

```diff
@@ -2,91 +2,40 @@
 core = 7.x
 
 api = 2
-projects[drupal][version] = "7.43"
+projects[drupal][version] = "7.x"
 
 projects[ctools][subdir] = "contrib"
-projects[ctools][version] = "1.9"
-
 projects[cacheexclude][subdir] = "contrib"
-projects[cacheexclude][version] = "2.3"
-
 projects[entity][subdir] = "contrib"
-projects[entity][version] = "1.6"
-
 projects[field_group][subdir] = "contrib"
-projects[field_group][version] = "1.5"
-
 projects[globalredirect][subdir] = "contrib"
-projects[globalredirect][version] = "1.5"
-
 projects[google_analytics][subdir] = "contrib"
-projects[google_analytics][version] = "2.1"
-
 projects[gss][subdir] = "contrib"
-projects[gss][version] = "1.11"
-
 projects[imagecache_token][subdir] = "contrib"
-projects[imagecache_token][version] = "1.0-rc2"
-
 projects[jquery_update][subdir] = "contrib"
-projects[jquery_update][version] = "2.7"
-
 projects[libraries][subdir] = "contrib"
-projects[libraries][version] = "2.2"
-
 projects[link][subdir] = "contrib"
-projects[link][version] = "1.4"
-
 projects[menu_block][subdir] = "contrib"
-projects[menu_block][version] = "2.7"
-
 projects[menu_expanded][subdir] = "contrib"
-projects[menu_expanded][version] = "1.0-beta1"
-
 projects[metatag][subdir] = "contrib"
-projects[metatag][version] = "1.13"
-
 projects[references][subdir] = "contrib"
-projects[references][version] = "2.1"
-
 projects[pathauto][subdir] = "contrib"
-projects[pathauto][version] = "1.3"
-
 projects[readonlymode][subdir] = "contrib"
-projects[readonlymode][version] = "1.2"
-
 projects[redirect][subdir] = "contrib"
-projects[redirect][version] = "1.0-rc3"
-
 projects[seckit][subdir] = "contrib"
-projects[seckit][version] = "1.9"
-
 projects[site_map][subdir] = "contrib"
-projects[site_map][version] = "1.3"
-
 projects[token][subdir] = "contrib"
-projects[token][version] = "1.6"
-
 projects[transliteration][subdir] = "contrib"
-projects[transliteration][version] = "3.2"
-
 projects[views][subdir] = "contrib"
-projects[views][version] = "3.13"
-
 projects[webform][subdir] = "contrib"
-projects[webform][version] = "3.24"
-
 projects[wysiwyg][subdir] = "contrib"
-projects[wysiwyg][version] = "2.2"
-
 projects[xmlsitemap][subdir] = "contrib"
-projects[xmlsitemap][version] = "2.2"
```

Comment:
This one we want to freeze - I better check the version. Or maybe it has been resolved.